### DISCUSSION TITLE
Geometric Forcast

### DISCUSSION THREAD

**Discussion by creuzerm** [tcarwash/home-assistant_noaa-space-weather#35] [Ideas]
Is the Geometric forecast of interest?  There is a text file available at https://services.swpc.noaa.gov/text/3-day-geomag-forecast.txt that we can parse and display if there is interest.

I'd be happy to contribute a pull request with this if there is interest. If there isn't interest, that means a different module for me.

**Comment by tcarwash** [comment]
I think that's definitely of interest, if you want to send a PR I'll merge it in

**Reply by creuzerm** [reply]
Sounds good, I know what I am doing some weekend soon!

**Reply by tcarwash** [reply]
Right now all of the API calls and data manipulation is being done in github.com/tcarwash/swpclib, if you add a method there and include it in the "get_standard" method the data is easy to access in the home assistant integration. 

You might also check if there's a json endpoint available that duplicates that geomagnetic forecast (I did some cursory searching and didn't see anything) that would reduce parsing that needs to be done. 

I look forward to seeing what you come up with!
